apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryConnection.Queries.cs

```diff
@@ -0,0 +1,247 @@
+        /// the table first.</param>
+        /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied.</param>
+        /// <returns>An asynchronous sequence of the rows within the table.</returns>
+        public PagedAsyncEnumerable<TableDataList, BigQueryRow> ListRowsAsync(string tableId, TableSchema schema = null, ListRowsOptions options = null)
+            => GetBigQueryClient().ListRowsAsync(GetProjectId(), GetDatasetId(), tableId, schema, options);
+
+        /// <summary>
+        /// Lists the rows within a table within this client's project specified by dataset ID and table ID, similar to a <c>SELECT * FROM ...</c> query.
+        /// This method just creates a <see cref="TableReference"/> and delegates to <see cref="ListRowsAsync(TableReference, TableSchema, ListRowsOptions)"/>.
+        /// </summary>
+        /// <param name="datasetId">The dataset ID. Must not be null.</param>
+        /// <param name="tableId">The table ID. Must not be null.</param>
+        /// <param name="schema">The schema to use when interpreting results. This may be null, in which case it will be fetched from
+        /// the table first.</param>
+        /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied.</param>
+        /// <returns>An asynchronous sequence of the rows within the table.</returns>
+        public PagedAsyncEnumerable<TableDataList, BigQueryRow> ListRowsAsync(string datasetId, string tableId, TableSchema schema = null, ListRowsOptions options = null)
+            => GetBigQueryClient().ListRowsAsync(datasetId, tableId, schema, options);
+
+        /// <summary>
+        /// Lists the rows within a table, similar to a <c>SELECT * FROM ...</c> query.
+        /// </summary>
+        /// <param name="tableReference">A fully-qualified identifier for the table. Must not be null.</param>
+        /// <param name="schema">The schema to use when interpreting results. This may be null, in which case it will be fetched from
+        /// the table first.</param>
+        /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied.</param>
+        /// <returns>An asynchronous sequence of the rows within the table.</returns>
+        public PagedAsyncEnumerable<TableDataList, BigQueryRow> ListRowsAsync(TableReference tableReference, TableSchema schema = null, ListRowsOptions options = null)
+            => GetBigQueryClient().ListRowsAsync(tableReference, schema, options);
+        #endregion
+    }
+
+    /// <summary>
+    /// Command options for configuring a sql query or DML statement.
+    /// You may create a new <see cref="BigQueryCommand" /> and set <see cref="BigQueryCommand.CommandOptions" />
+    /// to an instance of this class to perform a SQL query or DML statement.
+    /// </summary>
+    public sealed class SqlCommandOptions : BigQueryCommandOptions
```

Comment:
Could this be moved to a separate file? This was why I missed it before :)

Reply:
yes thats reasonable.  my initial intention was that each file would contain the related functionality whether autogenerated or not -- so you wouldnt have to know up front where to look -- but the class can certainly be moved.